internal/rewrite_action.go

```diff
@@ -156,22 +156,8 @@ func (t *ImageTemplate) Apply(originalImage name.Reference, rules *rewrite.Rules
 	var rewrites []*RewriteAction
 
 	// Tag or Digest
-	if t.TagTemplate != "" {
+	if t.TagTemplate != "" || t.DigestTemplate != "" {
 		tagged = true
-		if rules.Tag != "" && rules.Tag != originalImage.Identifier() {
```

Comment:
rewriting the tag or the digest are no longer supported in the code. We might bring back the tag support in the future if needed.